.changeset/odd-guests-beam.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": patch
+---
+
+fix: fix broken local Durable Object proxies
```

Comment:
NIT: this description is a bit enigmatic... could you provide a bit more description? I.e. the fact that previously if there was not a `cf` property on the request it was being rendered in the header as `undefined` that could not then be JSON parsed causing the "DO proxy "undefined" is not a valid JSON" error.

Reply:
sure I'll expand on this 🙂 

I didn't go too much in details since the breakage was all internal and not something related to the user code so I figured that keeping it generic in the changeset made sense 🤔

Reply:
I've updated the changeset, please have another look 🙏